### ISSUE TITLE
In ADDING_NEW_DATASETS.md, devs shouldn't have to re-install nycdb on every code change

### ISSUE THREAD

**Issue by toolness** [aepyornis/nyc-db#87]
[Step 6](https://github.com/aepyornis/nyc-db/blob/master/src/ADDING_NEW_DATASETS.md#step-6---re-install-nycdb-to-register-the-new-dataset) currently says:

> ### Step 6 - re-install nycdb to register the new dataset
>
> - `pip uninstall nycdb`
> - `pip setup.py install` (from inside of the `src` directory)

This is quite cumbersome to do after every code change and shouldn't be necessary if the developer has installed nycdb in [editable mode](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs), e.g. via `pip install -e`.  This is actually how the project is installed in the Docker setup, so at the very least, we should add a note mentioning that they won't need to perform this step if they're using Docker.  However, I think that ideally we should tell developers to install nycdb in editable mode if they actually want to hack on it, and then just remove step 6 entirely.

**Comment by aepyornis** [comment]
indeed! In the [src/Makefile](https://github.com/aepyornis/nyc-db/blob/master/src/Makefile#L7-L8)  which I use for whiling working on it, I use ` pip install -e . ` and it works great!
